newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/ExpressionAutocompletionsDisplayer.js

```diff
@@ -313,6 +313,7 @@ export default function ExpressionAutocompletionsDisplayer({
                     : undefined;
 
                   return expressionAutocompletion.kind === 'Text' ||
+                    expressionAutocompletion.kind === 'FullExpression' ||
```

Comment:
Can you explain why this is useful exactly?

Reply:
The autocompletion is not a text but an expression. It will suggest things like `"MyTimer" + VariableString(Index)`. It must not add quote around the autocompletion. I had to define a new kind of completion because of this.